pkg/config/v2/upstream.go

```diff
@@ -233,6 +233,7 @@ type ClusterManagerConfig struct {
 
 type ClusterManagerConfigJson struct {
 	TLSContext        TLSConfig `json:"tls_context,omitempty"`
+	ClusterPoolEnable bool      `json:"cluster_pool_enable,omitempty"`
```

Comment:
This configuration should also be supported in the cluster.
For example a scenario where http no needs pool but dubbo does.